operator.go

```diff
@@ -0,0 +1,149 @@
+package cloudconfig
+
+import (
+	"bytes"
+	"html/template"
+	"strings"
+
+	"github.com/giantswarm/awstpr"
+)
+
+var (
+	awsFilesMeta []FileMetadata = []FileMetadata{
+		FileMetadata{
+			AssetPath:   "templates/operator/aws/decrypt-tls-assets",
+			Path:        "/opt/bin/decrypt-tls-assets",
+			Owner:       "root:root",
+			Permissions: 0700,
+		},
+	}
+	awsUnitsMeta []UnitMetadata = []UnitMetadata{
+		UnitMetadata{
+			AssetPath: "templates/operator/aws/decrypt-tls-assets.service",
+			Name:      "decrypt-tls-certs.service",
+			Enable:    true,
+			Command:   "start",
+		},
+	}
+	kvmUnitsMeta []UnitMetadata = []UnitMetadata{
+		UnitMetadata{
+			AssetPath: "templates/operator/kvm/add-route-bridge.service",
+			Name:      "add-route-bridge.service",
+			Enable:    true,
+			Command:   "start",
+		},
+	}
+)
+
+type FileMetadata struct {
+	AssetPath   string
+	Path        string
+	Owner       string
+	Permissions int
+}
+
+type FileAsset struct {
+	Metadata FileMetadata
+	Content  []string
+}
+
+type UnitMetadata struct {
+	AssetPath string
+	Name      string
+	Enable    bool
+	Command   string
+}
+
+type UnitAsset struct {
+	Metadata UnitMetadata
+	Content  []string
+}
+
+type OperatorExtension interface {
+	GetFiles() ([]FileAsset, error)
+	GetUnits() ([]UnitAsset, error)
+}
```

Comment:
I like that interface. It makes sense for the idea of having an abstracted cloud config manager. If we like we can extend this in the future with authorized keys and things like this. Only two things toward that. 

- IMO there should not be anything AWS/KVM related inside of this project. I feel like my comments are unclear or something. We maybe should talk about this again. I do not understand why there is still code that only the AWS operator cares about in here. The implementation of the interface should be done inside of the AWS operator. 
- I would like to align the naming to the golang convention. Getters do not have the `get` prefix. See https://golang.org/doc/effective_go.html#Getters.